ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -164,6 +164,20 @@ export const AdminInstitutionEdit = fp.flow(withUrlParams(), withNavigation)(cla
     }
   }
 
+  async componentDidUpdate(prevProps: Props) {
+    // If institution short Name is passed in the URL get the institution details
+    if (hasNewValidProps(this.props, prevProps, [p => p.urlParams.institutionId])) {
+      const loadedInstitution = await institutionApi().getInstitution(this.props.urlParams.institutionId);
```

Comment:
since this block duplicates one in componentDidMount, we can extract a function

Reply:
this is true but also I'm going to delete it in #5450 so I don't care very muh